lua/feline/init.lua

```diff
@@ -8,6 +8,19 @@ local M = {}
 
 M.reset_highlights = gen.reset_highlights
 
+function M.update_all_windows()
+    for _, winid in ipairs(api.nvim_list_wins()) do
```

Comment:
Wrap the inside of the for loop, so it skips updating if the window is relative (floating).
i.e. ` if api.nvim_win_get_config(winid).relative == '' then ... end`

Reply:
Or you could filter them
i.e. `local windows = vim.tbl_filter(is_regular_window, api.nvim_tabpage_list_wins(0))`
where `regular_window` is a function that chck if the winid is relative.

I think maybe using `nvim_tabpage_list_wins` is better since we only care about the windows in the current tabpage, therefore reducing unnecessary computation.